case-study-hydropro-solutions.mdx

```diff
@@ -0,0 +1,72 @@
+---
+title: 'HydroPro Solutions achieves a 60% reduction in search time using the Xata Postgres-powered platform'
+description: 'Discover how HydroPro Solutions reduced search time by 60% and transformed their operations with the powerful Postgres-based serverless database solution from Xata, streamlining water meter installations and improving overall efficiency.'
+image:
+  src: https://raw.githubusercontent.com/xataio/mdx-blog/main/images/sava-labs-case-study@2x.png
+  alt: Case Study
+author: Cezzaine Zaher
+date: 09-18-2024
```

Comment:
Should be the 19th if publishing on Thursday

Reply:
Good catch, thank you!